src/components/project/query.helpers.ts

```diff
@@ -80,6 +80,18 @@ export const projectListFilter = (filter: ProjectFilters) => (query: Query) => {
     ]);
   }
 
+  if (filter.partnerId != null) {
+    query
```

Comment:
@CarsonF - Although this is working in my test cases, I'm not sure if this is the correct way to do this?

Reply:
I'd prefer to use a where clause path condition, like presetInventory above. Technically both work, but:
- using a where clause ensures rows don't expand.
   Row expansion happens if you like match multiple partners for a single project, so each row is now the combination of a unique project & partner leading to duplicate rows with the same project. This shouldn't happen here but it's something to keep in mind.
-  ensures clause siblings are ok.
    Like for example, you can't end a subquery and go straight into a where clause without having a `with` statement in-between.
https://github.com/SeedCompany/cord-api-v3/blob/e618a443ed797a7583bd87704842baead638de37/src/components/project/query.helpers.ts#L90-L92
   I think you can't have a where clause go straight into another match clause either. That was a problem here previously due to all of these being dynamic.
- Ultimately it's more idiomatic. We're trying to "filter data down based on these conditions" not "match related data".